containers/images/pulp/Containerfile.web.j2

```diff
@@ -1,13 +1,24 @@
 FROM {{ registry | default('quay.io') }}/{{ project | default('pulp') }}/{{ item.value.base_image_name | default('pulp') }}:{{ item.value.tag | default('latest') }} as builder
 
-RUN mkdir -p /etc/nginx/pulp
+RUN mkdir -p /etc/nginx/pulp \
+             /www/data
 {% for plugin in item.value.plugin_snippets %}
 RUN ln /usr/local/lib/python{{ item.value.python_version }}/site-packages/{{ plugin }}/app/webserver_snippets/nginx.conf /etc/nginx/pulp/{{ plugin }}.conf
 {% endfor %}
 
+{% if item.value.base_image_name == "galaxy" %}
+RUN cp -fr /usr/local/lib/python{{ item.value.python_version }}/site-packages/galaxy_ng/app/static/galaxy_ng/. /www/data
+{% endif %}
+
+
 FROM nginx:latest
 
-RUN mkdir -p /etc/nginx/pulp
+RUN mkdir -p /etc/nginx/pulp \
+             /www/data
 WORKDIR /etc/nginx/pulp
 COPY --from=builder /etc/nginx/pulp .
+{% if item.value.base_image_name == "galaxy" %}
+WORKDIR /www/data
```

Comment:
I don't understand why `/www/data`

Reply:
I just picked a commonly used location for nginx static files `/www/data`.

It lines up with this added default: https://github.com/pulp/pulp-operator/pull/91/files#diff-12419eb0d7211b069758a1c95d49c6567c1c01edf0fc3e73bf3470e9c6abeab9R30
which is used in the nginx.conf change here:
https://github.com/pulp/pulp-operator/pull/91/files#diff-786fc0ed06fd8ad3a8505e97e90c90bea9356f93255943f129901fcfa2866237R75-R77